src/ontology/config/icd11foundation-property-map.sssom.tsv

```diff
@@ -0,0 +1,5 @@
+subject_id	object_id
+http://id.who.int/icd/schema/isObsolote	owl:deprecated
+http://id.who.int/icd/schema/longDefinition	http://purl.org/dc/terms/description
+http://id.who.int/icd/schema/note	rdfs:comment
+skos:definition	IAO:0000115
```

Comment:
## `obo:IAO_0000115`
- [ ] Advise @matentzn  or @twhetzel 
I'm getting an `<obo:IAO_0000115 xml:lang="en">` tag. I was expecting `<IAO:0000115 xml:lang="en">`. I checked our other components though, and I see that the same thing is also happening for them.

Am I correct that this is unexpected? Is this potentially causing any problems for us? Should I create an issue or do anything in this PR?

Reply:
No problem you can leave this. In RDF prefixes are syntactic sugar.